cpu/sam0_common/periph/pwm.c

```diff
@@ -369,7 +368,9 @@ static void _tcc_set(Tcc *tcc, uint8_t chan, uint16_t value)
     /* TODO: use OTMX for pin remapping */
     chan %= _tcc_get_cc_numof(tcc);
     tcc->CC[chan].reg = value;
+#ifdef REV_TCC
     while (tcc->SYNCBUSY.reg & (TCC_SYNCBUSY_CC0 << chan)) {}
+#endif
```

Comment:
Doesn't this change behavior for non SAMD20 MCUs that do not decleare `REV_TCC`?

There previously `TC_SYNCBUSY_CC0 << chan` was checked instead of `TCC_SYNCBUSY_CC0 << chan` - now no synchronization is done.

Reply:
I must admit this is a bit hacky.
`_tcc_set()` can only called if TCC is present, this file relies heavily on compile-time evaluation of `_use_tcc()`/`_use_tc()` to avoid large `#ifdef` blocks.

I think there is a cleaner solution though.

Reply:
> I think there is a cleaner solution though.

I was wrong wrong.